src/sentry/api/endpoints/project_stacktrace_link.py

```diff
@@ -43,11 +43,6 @@ def get(self, request, project):
         # the ordering is deterministic
         configs = RepositoryProjectPathConfig.objects.filter(project=project)
 
-        # we only want to attempt the stack trace linking IF the user
-        # has a configuration set up, otherwise we don't care
-        if not configs:
```

Comment:
this condition is necessary since we return the same thing at the end of the function